content/en/tracing/trace_collection/custom_instrumentation/ios/otel.md

```diff
@@ -0,0 +1,341 @@
+---
+title: iOS and tvOS applications custom instrumentation using OpenTelemetry API
+kind: documentation
+code_lang: swift
+type: multi-code-lang
+code_lang_weight: 80
+description: 'Instrument your iOS and tvOS application with OpenTelemetry API, to send traces to Datadog.'
+aliases:
+- /tracing/trace_collection/otel_instrumentation/ios/
+- /tracing/trace_collection/custom_instrumentation/otel_instrumentation/ios
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+    - link: '/opentelemetry/guide/otel_api_tracing_interoperability'
+      tag: 'Documentation'
+      text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
+---
+
+{{% otel-custom-instrumentation %}}
+
+## Requirements and limitations
+
+<!-- TODO: version needs to corrected after release -->
+- DatadogTrace for iOS and tvOS version 1.0.0+.
+
+The following OpenTelemetry Swift features are implemented in the Datadog library as noted:
+
+| Feature | Support notes |
+|-|-|
+| [Span attributes][2] | Supported |
+| [Span events][3] | Unsupported |
+| [Span exporters][5] | Unsupported |
+| [Span links][1] | Supported |
+| [Span processors][4] | Unsupported |
+| Context propagation | [W3C Trace Context and Datadog header formats][6] are enabled by default.  |
+
+## Tracing iOS Applications with OpenTelemetry
```

Comment:
We should also add a reference from the main iOS tracing page to this one should people want to trace with Otel API.
[https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/ios](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/ios)

Reply:
Sure.